### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (#wcb_2)

### ISSUE THREAD

**Issue by aarongustafson** [aarongustafson/aaron-gustafson.com#108]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Hi! :: Aaron Gustafson](https://www.aaron-gustafson.com/)
#### Element path
#wcb_2
#### Snippet
`<span id="wcb_2">&nbsp;Cleaner than 99% of pages tested</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 1.31 (foreground color: #0e11a8, background color: #454545, font size: 11.3pt (15px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 138.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
